markdownlint.js

```diff
@@ -10,13 +10,10 @@ const process = require('process');
 const program = require('commander');
 
 const options = program.opts();
-const differenceWith = require('lodash.differencewith');
-const flatten = require('lodash.flatten');
+const glob = require('glob');
 const markdownlint = require('markdownlint');
 const rc = require('run-con');
-const glob = require('glob');
```

Comment:
minor, but you might want to move the require back to the same position to reduce the diff.
Doesn't look like there was a particular order to start with though

Reply:
I agree in general, but this one seems harmless.